components/ApplicationGrantSelector/ApplicationGrantSelector.jsx

```diff
@@ -0,0 +1,119 @@
+import Router from 'next/router';
+import { isExpired } from '../../utils/date';
+import { stepKeys } from '../Steps';
+
+const ApplicationGrantSelector = ({ date, expirationDate }) => {
+  return (
+    <>
+      <OhlgGrantSelector date={date} expirationDate={expirationDate} />
+      <ArgGrantSelector date={date} expirationDate={expirationDate} />
+    </>
+  );
+};
+
+const OhlgGrantSelector = ({ date, expirationDate }) => {
+  return (
+    <>
+      <h1 data-testid="home-page-heading">
+        Apply for the Omicron Hospitality and Leisure Grant
+      </h1>
+      <p className="govuk-body">Some text here</p>
+      {(!expirationDate ||
+        !isExpired(new Date(expirationDate), new Date(date))) && (
+        <button
+          href="#"
+          role="button-ohlg"
+          draggable="false"
+          className="govuk-button govuk-button--start govuk-!-margin-top-3"
+          data-module="govuk-button"
+          onClick={() => Router.push(`/grant/ohlg/step/${stepKeys[0]}`)}
```

Comment:
This is a weird pattern, surely we would just use a proper `<a>` tag here? I think I saw this pattern elsewhere in the codebase, so I see where it's come from. Feels like a good moment to fix it (and probably use `<Link>` from [`next/link`](https://nextjs.org/docs/api-reference/next/link) too?